### DISCUSSION TITLE
Should we start migrating the wiki?

### DISCUSSION THREAD

**Discussion by MrMagician** [TeamGalacticraft/Galacticraft#233] [Meta]
I wanted to contribute to my favorite mod, So I looked around and noticed that the wiki on this repository was empty.
Does anyone mind if I start migrating a few pages from [the official wiki](https://wiki.micdoodle8.com) to this repository's wiki?

**Comment by EzerArch** [comment]
What kind of content would you like to migrate, only the codding part?

**Comment by MrMagician** [comment]
The main page, And some useful pages, like the oil one. When you mean the coding part. Do you mean the html that the pages are made from? Maybe i'm not understanding.

**Comment by JustPyrrha** [comment]
its probably a good idea to have some sort of Galacticraft 5 specific wiki but I'd like to put in some effort to do it right.
It'll probably be mkdocs with a custom extensions for models and recipes.
This'll be because we want it to be accessible as possible to contributions from the community and markdown is a good way to do that.

**Reply by JustPyrrha** [reply]
I've got a meeting for work but after that I'll take a look and see what we can sort out.

**Comment by MrMagician** [comment]
> It'll probably be mkdocs with a custom extensions for models and recipes.

While we're at it, Should I look around for a more space like mkdocs theme?

**Reply by JustPyrrha** [reply]
a deep blue or dark purple colour scheme in the mkdocs material theme would work

**Comment by MrMagician** [comment]
[Here is one I found](https://github.com/michaeltlombardi/mkdocs-psinder), But I don't think the PowerShell theme matches well.
